tools/mason/MasonNew.chpl

```diff
@@ -119,14 +119,16 @@ proc beginInteractiveSession() throws {
       if gotCorrectPackageName == false {
         write("Package name: ");
         IO.stdout.flush();
-        readf("%s", packageName);
+        IO.stdin.readline(packageName);
+        packageName = packageName.strip();
         if validatePackageName(packageName) == true then
           gotCorrectPackageName = true;
       }
       if gotCorrectPackageVersion == false {
         write("Package version (0.1.0): ");
         IO.stdout.flush();
-        readf("%s", version);
+        IO.stdin.readline(version);
+        version = version.strip();
         if version == "" then version = "0.1.0";
         checkVersion(version); 
         gotCorrectPackageVersion = true;
```

Comment:
If an user is using `mason new --interactive` for creating their project, I assumed that the version should always be `0.1.0`. However, I haven't added a check to force the user to use `0.1.0`. Should there be a check ?

Reply:
I don't think so. I think `0.1.0` makes sense as a default, but may a user has a reason to start with a different version.